src/UserFrontend.php

```diff
@@ -61,13 +61,22 @@ public static function login_head() {
    */
   public static function login_form_defaults(array $args) {
     $args['value_remember'] = TRUE;
+    $args['remember'] = FALSE;
     return $args;
   }
 
   /**
    * @implements login_enqueue_scripts
    */
   public static function login_enqueue_scripts() {
+    echo <<<EOD
+<style>
+#loginform .forgetmenot {
+  display: none;
+}
+</style>
```

Comment:
Wondering whether the style should be added to the existing default CSS below?

Existing sites with a custom `login.css` file in their theme would remain to have the remember_me checkbox visible then – which sounds appropriate to me.